nodelibs/deno/worker_threads.ts

```diff
@@ -0,0 +1,127 @@
+/// <reference lib="webworker"/>
+
+import { EventEmitter } from 'https://deno.land/std@0.105.0/node/events.ts';
+
+function unimplemented(name: string) {
+  throw new Error(
+    `Node.js worker_threads ${name} is not currently supported by JSPM core in Deno`,
+  );
+}
+
+const environmentData = new Map();
+let threads = 0;
+
+interface _WorkerOptions extends WorkerOptions {
+  workerData?: any;
+  resourceLimits?: {
+    maxYoungGenerationSizeMb?: number;
+    maxOldGenerationSizeMb?: number;
+    codeRangeSizeMb?: number;
+    stackSizeMb?: number;
+  };
+}
+interface _Worker extends Worker, EventEmitter {}
+class _Worker extends Worker {
+  public threadId: number;
+  public resourceLimits: Required<NonNullable<_WorkerOptions['resourceLimits']>>; 
+
+  constructor(specifier: URL | string, options?: _WorkerOptions) {
+    if (options == null) options = { type: 'module' };
+    else if (typeof options === 'object') options.type = 'module';
+    super(specifier, options);
```

Comment:
I think this should be:

```suggestion
    super(specifier, options);
    EventEmitter.call(this);
```

I am actually quite surprised if it was working since in Deno the constructor sets up `this._events = Object.create(null);` which seems pretty important to the implementation...

Reply:
There is another problem, `EventEmitter` is defined with `class`, so `EventEmitter.call` will not work.